### PR TITLE
Fix add and delete  command button in /admin/bots crashes

### PR DESCRIPTION
**Pull request by kimenyikevin**
## Summary

A summary of changes being made in this PR

## Checklist
- [x] Pre-push checks pass `npm run check`
  - [x] Linter passing via `npm run lint`
  - [x] Unit & Integration tests passing via `npm run test:packages`
  - [x] Docker build process passing via `npm run build-client`
- [x] If this PR is still a WIP, convert to a draft 
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References

References to pertaining issue(s)

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

@HexaField @speigg @NateTheGreatt
## Have the following task to be complete 
- Add confirm message before delete bot and fix refresh manually after delete bot
- Merge all style in one file and define re-usable component 
- Refactor update bot 
- Fix remove and add command to bot

### CODE REVIEW

File: packages/client-core/src/admin/common/InputSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          name={name}
+          displayEmpty
+          className={classes.select}
+          MenuProps={{ classes: { paper: classes.selectPaper } }}
+        >
+          <MenuItem value="" disabled>
+            <em>Select {name}</em>
+          </MenuItem>
+          {menu.map((el) => (
+            <MenuItem value={el.value} key={el.value}>
```
Comment (zulqarnainhanif) [inline packages/client-core/src/admin/common/InputSelect.tsx:36]: use index for key prop

**Comment by zulqarnainhanif** [comment]
@kimenyikevin everything is working fine, my only concern is should we allow adding multiple commands with same names. what are your thoughts on that @HexaField ?

File: packages/client-core/src/admin/common/InputSelect.tsx (hunk 1 above)
Comment (zulqarnainhanif) [reply packages/client-core/src/admin/common/InputSelect.tsx:36]: Reason:
Screenshot 2022-01-26 at 10 02 20 PM

**Comment by HexaField** [comment]
Let's stay on the safe side an disallow it until we have a better idea of how we want the bots to work

**Review by zulqarnainhanif — APPROVED** [review]

**Review by zulqarnainhanif — APPROVED** [review]

**Pull request merged**